src/main/java/ch/njol/skript/classes/data/DefaultComparators.java

```diff
@@ -103,6 +104,8 @@ public boolean supportsOrdering() {
 
 			@Override
 			public Relation compare(Slot o1, Slot o2) {
+				if ((o1 instanceof EquipmentSlot && !(o2 instanceof EquipmentSlot)) || !(o1 instanceof EquipmentSlot) && o2 instanceof EquipmentSlot)
```

Comment:
Maybe this would be better of as:
```suggestion
				boolean equipment1 = o1 instanceof EquipmentSlot;
				boolean equipment2 = o2 instanceof EquipmentSlot;
				if (equipment != equipment2)
```
Not sure about those variable names though.

Reply:
kinda just does the same thing with more lines.

Reply:
Maybe - I mainly suggested this just to avoid the double instanceof checks. Not that it really has that much of an impact though

Reply:
in all fairness your suggestion is a double instance check anyways

Reply:
updated 👍